pkg/webui/console/views/application-expdata/index.js

```diff
@@ -128,6 +145,7 @@ const ApplicationDataExport = () => {
   }
 
   const fetchData = () => {
+    set(true);
```

Comment:
what does the set(true) do here?

Reply:
is it starting the validation?

Reply:
It seemed like I didn't fully delete the set(true) when I was messing around with some other forms of validation, but it is now deleted as it is unnecessary.